.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 16.0.0
```

Comment:
Not sure if we want to be committing this in. What is this file exactly?

Reply:
That was a mistake. Good catch. Will remove